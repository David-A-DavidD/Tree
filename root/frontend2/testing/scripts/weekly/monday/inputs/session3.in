login
admin
refund
buyUser
sellUser
100
logout
exit