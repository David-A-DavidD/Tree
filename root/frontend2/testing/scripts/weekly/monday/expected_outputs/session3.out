Enter transaction code (or 'exit' to quit): Enter username: Login successful.
Enter the buyer's username: Enter the seller's username: Enter the amount of credit to transfer: Refund successful. Transferred 100 credits from sellUser to buyUser.
Logout successful.